docs/public/replication/asset-deleted.md

```diff
@@ -0,0 +1,30 @@
+# Replicate deleted Asset
+
+**Actor**: Peregrine CMS Author  
+**Description**: Author deletes an Asset.
+
+## Preconditions:
+1. Asset already exists in the system.
+1. Asset is already replicated.
+
+## Postconditions:
+1. The Asset is now removed from replication cache.
+1. All unreferenced tags associated with the Asset
+   are now removed from replication cache.
+
+## Main Course:
+1. Author deletes an Asset.
+1. System presents the author with a confirmation dialog asking about removal of
+   replicated resources.
+1. System presents the author a list of other pages that reference the Asset to be removed.
```

Comment:
There may be a gap between what is said and what is meant by this line. Seems to say that a list pages, which use the asset, will be removed. Perhaps what is meant is asset references will be removed from a list of pages.

Reply:
@cmrockwell - You are correct. The pages will NOT be removed. The question is really this: 
1. Should we actually remove the references from the various nodes?
2. Or, should we simply inform the user that the list of pages will be impacted if this action is executed?

CC: @reusr1